ts/features/zendesk/components/ZendeskSupportComponent.tsx

```diff
@@ -128,21 +133,25 @@ const ZendeskSupportComponent = (props: Props) => {
           {I18n.t("support.helpCenter.cta.contactSupport")}
         </Label>
       </ButtonDefaultOpacity>
-      <View spacer={true} />
-      <ButtonDefaultOpacity
-        onPress={() => {
-          showSupportTickets();
-          workUnitCompleted();
-        }}
-        style={{
-          alignSelf: "stretch"
-        }}
-        disabled={false}
-        bordered={true}
-        testID={"showTicketsButton"}
-      >
-        <Label>{I18n.t("support.helpCenter.cta.seeReports")}</Label>
-      </ButtonDefaultOpacity>
+      {showAlreadyOpenedTicketButton && (
+        <>
+          <View spacer={true} />
+          <ButtonDefaultOpacity
+            onPress={() => {
+              showSupportTickets();
+              workUnitCompleted();
+            }}
+            style={{
+              alignSelf: "stretch"
+            }}
+            disabled={false}
+            bordered={true}
+            testID={"showTicketsButton"}
+          >
+            <Label>{I18n.t("support.helpCenter.cta.seeReports")}</Label>
+          </ButtonDefaultOpacity>
```

Comment:
```suggestion
          </ButtonDefaultOpacity>
          <View spacer={true} />
```

The same of the previous comment, to add some padding on Android.

Reply:
Thank you, added in [9906619](https://github.com/pagopa/io-app/pull/3605/commits/99066191147bbaca59493ea60b3cce34e563f4b2)